corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -329,7 +329,8 @@ def as_string(self):
             blob.close()
 
     def get_http_response(self):
-        return stream_response(self.blobdb.get(self.get_filename()))
+        headers = {'Content-Length': self.blobdb.size(self.get_filename())}
```

Comment:
it's true, the content-length is returned when we get the object. i wasn't sure how to incorporate that without changing the api pretty significantly. i guess we can use some caching. or return a BlobInfo object on `get`

i think we can definitely use XSendfile. we already do for some things. on my roadmap for phase 2. essentially just wanted to get this working as soon as possible to alleviate the disk space issue we're having on icds